Gemfile

```diff
@@ -11,8 +11,14 @@ gem 'sinatra-activerecord'
 gem 'puma'
 gem 'tux'
 
+# These gems are only installed when RACK_ENV is either `development` or `test`
```

Comment:
Actually `bundle install` doesn't look at `RACK_ENV` or `RAILS_ENV` but depends on its own `--without` flag.

To avoid installing the pg gem in development:

```
bundle install --without production
```

If this might be problematic because students aren't really trained to invoke it that way yet and might not have dependencies for the `pg` gem then it might be better left commented out, so they can add it later if they are deploying for pg in production.